addons/im_livechat/models/discuss_channel_member.py

```diff
@@ -18,7 +18,7 @@ def _gc_unpin_livechat_sessions(self):
             ('channel_id.channel_type', '=', 'livechat'),
         ])
         sessions_to_be_unpinned = members.filtered(lambda m: m.message_unread_counter == 0)
-        sessions_to_be_unpinned.write({'is_pinned': False})
+        sessions_to_be_unpinned.write({'unpin_dt': datetime.now()})
```

Comment:
wondering about consistency, most other places use `fields.Datetime.now()`, I think it's a better version for us?